src/server/replica.cc

```diff
@@ -1023,7 +1030,21 @@ Replica::Info Replica::GetInfo() const {
     res.full_sync_done = (state_mask_.load() & R_SYNC_OK);
     res.master_last_io_sec = (ProactorBase::GetMonotonicTimeNs() - last_io_time) / 1000000000UL;
     return res;
-  });
+  };
+
+  if (ABSL_PREDICT_TRUE(Sock()))
+    return Proactor()->AwaitBrief(f);
+  else
+    return ProactorBase::me()->AwaitBrief(f);
+  /**
+   * when this branch happens: there is a very short grace period
+   * where Sock() is not initialized, yet the server can
+   * receive ROLE/INFO commands. That period happens when launching
+   * an instance with '--replicaof' and then \b immediately
```

Comment:
\b? mistake?

Reply:
\b makes text bold in VScode. I'll remove it to avoid confusion!